tests/figures/chart/chart_plugin.test.ts

```diff
@@ -2960,4 +2967,56 @@ describe("trending line", () => {
       expect(value).toEqual(expectedValue);
     }
   });
+
+  test("trend line is bypassed without sufficient dataset values", () => {
+    const model = new Model();
+    setCellContent(model, "A1", "test");
+    createChart(
+      model,
+      {
+        type: "line",
+        dataSets: [{ dataRange: "A1", trend: { type: "polynomial", order: 1, display: true } }],
+        fillArea: true,
+      },
+      "chartId"
+    );
+    let runtime = model.getters.getChartRuntime("chartId") as any;
```

Comment:
```suggestion
    let runtime = model.getters.getChartRuntime("chartId") as LineChartRuntime;
```

or do we need to do ts-ignore on the rest of the test if we do that  ?

Reply:
We do need the ts ignore because not all runtimes have `chartJsConfig` :/